.github/workflows/build.yml

```diff
@@ -44,10 +45,13 @@ jobs:
       run: nix-instantiate --eval -E '(import <nixpkgs> {}).lib.version'
     - name: Setup cachix
       uses: cachix/cachix-action@v10
+      # Don't replace the template string here!
```

Comment:
I'd be even more explicit: "Don't replace '<YOUR_CACHIX_NAME>' here!".

Or we could avoid the template string entirely and use a null/empty string

Reply:
I think the checks need to keep the template strings because otherwise the jobs would run for someone who just generated a repo from the template.

Reply:
Yes we'd have to keep the checks (`!= '<YOUR_CACHIX_NAME>'` -> `!= ''` or something). Opinions @Mic92 ?